includes/functions.php

```
@@ -11,5 +13,45 @@ function str_contains($haystack, $needle) {
     }
 }
 
+// return the value of the requested call type.
+if( !function_exists( 'get_call_types' ) ) {
+  function get_call_types( $type ) {
+    static $call_types;
+
+    $call_types = API\RETS_API::CALL_TYPES;
+    if( isset( $call_types[ $type ] ) ) {
+      return $call_types[ $type ];
+    }
+
+    return false;
```

Unless you have a good reason not to, I think this function should throw an exception if the call type is not set. The only cause of that problem would be a developer spelling a call type wrong, in which case the error is either thrown here or when they pass `false` to a function expecting a call type. Since their error would have occurred when calling this function, the exception should be thrown here.

---

@NateLanza function removed.